ci.docker-compose-no-metrics.yml

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
instead of another docker-compose file to maintain, can we make use of multiple env files or just set the env variable directly? seems like the only difference should be setting `METRIC_DATABASE_ENABLED` to `false`

Reply:
@pirtleshell I think database container is removed from here, but I guess we can just launch it with:
```
docker compose up redis, proxy
```
instead of creating new file

Reply:
Refactored. As there is some problem with passing env variable into the container passing them as part of .env and no_metric.env files depending on type of tests running. Keep one docker compose for both ci